ProcessHacker/mainwnd.c

```diff
@@ -416,7 +416,7 @@ VOID PhMwpInitializeControls(
     PhMwpProcessTreeNewHandle = CreateWindow(
         PH_TREENEW_CLASSNAME,
         NULL,
-        WS_CHILD | WS_VISIBLE | WS_CLIPCHILDREN | WS_CLIPSIBLINGS | WS_BORDER | TN_STYLE_ICONS | TN_STYLE_DOUBLE_BUFFERED | TN_STYLE_ANIMATE_DIVIDER | thinRows,
```

Comment:
You should make this an optional setting. 

1) Open \ProcessHacker\settings.c and add a new setting: PhpAddIntegerSetting(L"TreeListBorderEnable", L"1");

2) Instead of removing WS_BORDER use the following code: 
ULONG treelistBorder;

treelistBorder = PhGetIntegerSetting(L"TreeListBorderEnable") ? WS_BORDER : 0;

mainwnd.c: change "| thinRows" to "| treelistBorder | thinRows"

Reply:
Ah, right. Didn't realize that.
Thanks for pointing out.